compiler/resolution/generics.cpp

```diff
@@ -272,6 +261,20 @@ instantiate_tuple_autoCopy(FnSymbol* fn) {
   normalize(fn);
 }
 
+static void
```

Comment:
Why aren't the tuple initcopy and autocopy functions named in the same way as all others?  That's probably a question to be answered by a future patch, though.